en/components/grids_templates/cell-editing.md

```diff
@@ -162,15 +162,29 @@ Please notice that if the cell you want to update, via reference to [`IgxGridCel
 
 You can see and learn more for default cell editing templates in the [general editing topic](editing.md#editing-templates).
 
-If you want to provide a custom template which will be applied when a cell is in edit mode, you can make use of the [`igxCellEditor` directive]({environment:angularApiUrl}/classes/igxcelltemplatedirective.html). To do this, you need to pass an `ng-template` with the `igxCellEditor` directive and properly bind your custom control to the [`cell.editValue`]({environment:angularApiUrl}/classes/igxcellcomponent.html#editvalue):
+If you want to provide a custom template which will be applied when a cell is in edit mode, you can make use of the [`igxCellEditor` directive]({environment:angularApiUrl}/classes/igxcelltemplatedirective.html). To do this, you need to pass an `ng-template` marked with the `igxCellEditor` directive and properly bind your custom control to the [`cell.editValue`]({environment:angularApiUrl}/classes/igxcellcomponent.html#editvalue):
 
```

Comment:
Please add here something like the following explanation: 

> Keep in mind that by using `igxCellEditor` with `igx-select` component the keyboard navigation flow will be disrupted. The same applies for direct editing of the custom cell put in edit mode. This is because the focus will remain on the actual cell, not on the igx-select that we've added. This is why we are taking leverage of our `igxFocus` directive, which will move the focus directly in the igx-select and will help keeping a fluent editing flow of the cell/row.

@valadzhov, could you please add the same paragraph to the first sample, we are using igx-combo there, again with igxFocus directive. I would want to have this kind of paragraph there as well, if possible?

Reply:
The comments were addressed in [this PR](https://github.com/IgniteUI/igniteui-docfx/pull/2557).